templates/haproxy-reload

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# This script checks whether a reload of haproxy was requested, regenerates its
+# configuration file and reloads it.  The script is run via cron every minute.
+# This ensures that haproxy is reloaded at most once a minute.
+
+set -e
+
+# Name of file that indicates haproxy needs to be reloaded.
+HAPROXY_RELOAD_FILE="{{ LOAD_BALANCER_RELOAD_FILE }}"
+
+if [ -f "$HAPROXY_RELOAD_FILE" -o "$1" = "-f" ]; then
+    rm "$HAPROXY_RELOAD_FILE"
```

Comment:
This fails when the `$HAPROXY_RELOAD_FILE` doesn't exist but the `-f` flag has been passed (happens on the last step of the role).

To get around this temporarily I just surrounded the line with it's own file check:

```sh
if [ -f "$HAPROXY_RELOAD_FILE" ]; then
    rm "$HAPROXY_RELOAD_FILE"
fi
```